installer.py

```diff
@@ -121,22 +119,21 @@ def chooseVersion():
     QUESTION_TEXT = f"Which version of {config.FLPRODUCT} should the script install?"
     if config.TARGETVERSION is None or config.TARGETVERSION == "":
         options = ["10", "9", "Exit"]
-        versionChoice = curses_menu(options, TITLE, QUESTION_TEXT)
+        config.TARGETVERSION = curses_menu(options, TITLE, QUESTION_TEXT)
     else:
         versionChoice = config.TARGETVERSION
     logging.info(f"Target version: {config.TARGETVERSION}")
```

Comment:
It looks like the `else` statement can go away here, if there's no need for the intermediate variable `versionChoice`.

Reply:
Looks like the config setters below these lines can also go.

Reply:
Actually, there is a possibly of the user "exiting" here, so I probably need to do the reverse and have the curses menu set versionChoice.